CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Accord Project Cicero Word Add-in Contribution Guide
```

Comment:
Provide a link to this guide in the README as well.

Reply:
Okay 👍🏻

Reply:
@algomaster99, as suggested there is already a hyperlink present for these files in the readme in [this](https://github.com/accordproject/cicero-word-add-in#contributing) section so I don't think there is a need to provide this link at top of readme again. Still, I'm willing to know what are your views on this

Reply:
Oh okay. We won't need to add link in the README then if it already exists.

Reply:
Yeah, So I will update the readme by deleting the [`For Developers`](https://github.com/accordproject/cicero-word-add-in#for-developers) section

Reply:
@algomaster99, I had deleted the [ `For Developers` ](https://github.com/accordproject/cicero-word-add-in#for-developers) section now the readme looks like [this](https://github.com/Aniruddha-Shriwant/cicero-word-add-in/tree/docs-contributing#readme). We may update the readme again by including some more information related to the add-in, similar to `accordproject/template-studio` Readme's [`For Contributors`](https://github.com/accordproject/template-studio#for-contributors) section. 
For example:



# For contributors

The Cicero Word Add-in is a work in progress and we are very interested to hear from you. Let us know what you like or don't like about the Add-in, report bugs, or suggest new features by opening a GitHub issue, and Consider contributing to the project as we are very contributory-friendly! 🙂

---